### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by fridgepoet**
Don't know if that's correct, but noticed it as I was looking at the fill value options.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -9,7 +9,6 @@ export enum FormatOptions {
 export enum FillValueOptions {
   Previous,
   Null,
-  Value,
 }
```
Comment (andresmgot) [inline src/types.ts:12]: you can delete this one as well and use the  `FillValueOptions` from the SDK:

```
import { FillValueOptions } from '@grafana/aws-sdk/dist/sql/QueryEditor/FillValueSelect';
```

**Review by andresmgot — COMMENTED** [review]
Right, this is now defined in the SDK. You can still delete more things :)

**Review by andresmgot — APPROVED** [review]

File: src/types.ts [hunk 2]
```diff
@@ -1,17 +1,12 @@
 import { DataSourceSettings, SelectableValue } from '@grafana/data';
 import { AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData, SQLQuery } from '@grafana/aws-sdk';
+import { FillValueOptions } from '@grafana/aws-sdk/dist/sql/QueryEditor/FillValueSelect';
```
Comment (andresmgot) [inline src/types.ts:3]: if you increase the version of the sdk and merge it, a new release is automatically generated so you could use the nice import path here

Comment (fridgepoet) [reply src/types.ts:3]: https://github.com/grafana/grafana-aws-sdk-react/pull/11

It doesn't seem to build with that explicit path, by the way. Would you happen to know why?

Comment (andresmgot) [reply src/types.ts:3]: looks like some misconfiguration of jest: https://stackoverflow.com/questions/50863312/jest-gives-cannot-find-module-when-importing-components-with-absolute-paths

It should go away with the changes in the new version.

BTW, you also need to update `src/QueryEditor.test.tsx` (FillValueOptions is also used there)

**Pull request merged**